catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/SubmitStandingQueryRequestImpl.java

```diff
@@ -411,17 +410,20 @@ public void run() {
                 }
 
                 if (startDate != null) {
-                    if (startDate.getTime() > lastExecutionTime) {
-                        shouldRun = false;
+                    long now = System.currentTimeMillis();
+                    if (startDate.getTime() > now) {
+                        long waitToStart = startDate.getTime() - now;
+                        long waitSecs = waitToStart/1000;
```

Comment:
We can use TimeUnit.MILLISECONDS.toMinutes() here to make this easier to read and eliminate the constant.